### ISSUE TITLE
addon radio de v 1.1.3 deprecated ListItem.setInfo

### ISSUE THREAD

**Issue by EdMartens** [fivebanger/kodi_addons#8]
Hi,
Thanks for the great work. v1.1.3 solves a lot of compatible issues.
But is produces still warning about deprecated settings:

Setting most music properties through ListItem.setInfo() is deprecated and might be removed in future Kodi versions. Please use the respective setter in InfoTagMusic.
Using RPI3b and libreelec 12.01

Some improvement for v1.1.4 ;-)

**Comment by fivebanger** [comment]
Thanks for reporting. I'm aware of these warnings but since they do not harm at the moment I have ignored them. I'll fix those sooner or later....

**Comment by fivebanger** [comment]
fixed with v1.1.7

**Comment by fivebanger** [comment]
Breaks compatibility with Kodi < v20.x

**Comment by Ronny-nerd** [comment]
> Bricht die Kompatibilität mit Kodi <v20.x

That's right...regarding Kodi Matrix 19.5 ...but with your new version 1.1.11 of the addon the problem has been solved....thank you.

**Comment by fivebanger** [comment]
Fixed with v1.1.11
